cmd.md

```diff
@@ -0,0 +1,94 @@
+# Runtime Command Line Interface
+
+This document specifies the command line syntax for runtime operations defined in the [Runtime and Lifecycle](runtime.md) specification.
+It is STRONGLY RECOMMENDED that implementations adhere to the command line definition defined below to ensure interoperability.
+This document will not duplicate the information/semantics specified in the [Runtime and Lifecycle](runtime.md) document, rather it just focuses on syntax of the command line.
+
+## Convention
+
+For the purpose of this document, the following syntactical conventions are used:
+
+* `[ ... ]` denotes an optional field
+* `< ... >` denotes a substitution field. 
+The word(s), or phrase, within the `<>` describes the information to be passed-in.
+
+## General Format and Behavior
+
+The general format of all commands MUST be:
+```
+runtime [global-options] action [action-specific-options] [arguments]
+```
+
+Upon successful running of an action, the exit code MUST be zero.
+
+If there is an error during the running of an action, then:
+* the exit code MUST be non-zero
+* any error text MUST be displayed on stderr
+* the state of the environment MUST be the same as if the action was never attempted (modulo any possible trivial ancillary changes such as logging)
+
+### Global Options
+
+Global options are ones that apply to all actions.
+This specification doesn't define any global options.
+Implementation MAY define their own.
+
+### Action Specific Options
+
+All actions MUST support the `--help` action-specific-option, which:
+* MUST display some help text to stdout
+* MUST NOT perform the `action` itself
+* MUST have an exit code of zero if the help text is successfully displayed
+
+Implementations MAY define their own action-specific options.
+
+## Actions
+
+This section defines the actions defined by this specification.
+
+### State
+
+Format: `runtime [global-options] state [options] <container-id>`
+
+Options: None
+
+This action MUST display the state of the specific container to stdout.
```

Comment:
#513 has some wording around the [expected encoding][1] for the output from these commands.  Background on this in wking/oci-command-line-api#2 and wking/oci-command-line-api#16.

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files#diff-0c1e5a5baa5649945697a530e09acbf5R18